specification/marketplace/resource-manager/readme.md

```diff
@@ -44,28 +57,38 @@ APIs must only be added to this section when the API is publicly available in at
 input-file:
 - Microsoft.Marketplace/stable/2020-01-01/Marketplace.json
 - Microsoft.Marketplace/stable/2020-12-01/Marketplace.json
+```
 
 # Needed when there is more than one input file
 override-info:
   title: MarketplaceManagementClient
 ```
 
-### Tag: package-2019-12-01
+### Tag: package-2020-01-01
```

Comment:
So we are removing a swagger file and the corresponding tag here, I wonder why the breaking change CI does not report this?

Reply:
I was also wondering about that.  
In any case, this API was never published in this version so this is just a cleanup.